.gitignore

```diff
@@ -1 +1,4 @@
 node_modules/
+map.js
+style.css
+index.html
```

Comment:
Not sure. I saw them pop up but I thought I told Git to ignore those files.
I didn't edit them. :/
On Jan 18, 2016 4:11 PM, "Wouter" <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/woubuc/wu-map/pull/2#discussion_r50044388>:
>
> > @@ -1 +1,4 @@
> >  node_modules/
> > +map.js
> > +style.css
> > +index.html
>
> @Nuinethir <https://github.com/Nuinethir> why were these files added to
> gitignore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/woubuc/wu-map/pull/2/files#r50044388>.
>